packages/react/menu/src/Menu.tsx

```diff
@@ -1,23 +1,24 @@
 import * as React from 'react';
-import { RemoveScroll } from 'react-remove-scroll';
-import { hideOthers } from 'aria-hidden';
 import { composeEventHandlers } from '@radix-ui/primitive';
 import { createCollection } from '@radix-ui/react-collection';
 import { useComposedRefs, composeRefs } from '@radix-ui/react-compose-refs';
 import { createContextScope } from '@radix-ui/react-context';
+import { useDirection } from '@radix-ui/react-direction';
```

Comment:
Sorry, I reordered the imports here to match the alphabetical order in package.json, the only change is the addition of the slot import.